library/database/class.databasestructure.php

```diff
@@ -373,11 +373,41 @@ public function primaryKey($Name, $Type = 'int') {
     /**
      * Send a query to the database and return the result.
      *
+     * @deprecated
```

Comment:
`@deprecated since 2.3. Was incorrect public. Replaced by executeQuery().`

(To answer the obvious question: when referencing version numbers in code, use the next open source release number that your change may appear in)